docs/decisions/0007-executive-education-2U-tos-profile-data-route.md

```diff
@@ -0,0 +1,74 @@
+# 0007. Add page route for gathering consent for ToS and basic user profile data for Executive Education (2U) courses
+
+## Status
+
+Accepted (08-01-2022)
+
+## Context
+
+As part of our Product Line Integration work (PLI) to make 2U content (i.e., Executive Education courses) accessible to customers with an external Learning Management System (LMS), we will need to collect required information from the user in order for fulfillment of Executive Education (2U) courses.
+
+The fulfillment of Executive Education (2U) courses is handled by the GetSmarter Enterprise API Gateway (GEAG), in which edX for Business is considered an external service provider to GetSmarter. 
+
+The GEAG `/allocations` API endpoint denotes the following required information from the user:
+
+* Reading and agreeing to the Terms of Service (ToS)
```

Comment:
should we call this Terms and Conditions (T&C) throughout? to be consistent with what geag refers to them as

Reply:
Good call out. I'll rename these to Terms and Policies since GEAG returns studenttermsAndConditions, websiteTermsOfUse, privacyPolicy, and cookiePolicy.